inspec.yml

```diff
@@ -225,3 +225,9 @@ inputs:
     value:
       acronym: "DoD"
       full_form: "Department of Defense"
+
+  # V-259740
+  - name: min_org_allowed_postgres_version
+    desc: "The minimum Postgres version allowed by the organization"
+    type: string

```

Comment:
Is it easier / more intuitive for this to be a number instead of a string?

Reply:
I think it's more intuitive to use numbers, but I noticed some other variables tend to use strings with numeric values that don't have any math operations done against them, and matchers like 'be' seem happy making comparisons using strings for numbers. Wasn't sure if it was a standard practice to prefer strings for inspec controls.